baiji/config.py

```diff
@@ -1,7 +1,7 @@
 import os
 from baiji.exceptions import AWSCredentialsMissing
 
-class Credentials(object):
+class Settings(object):
```

Comment:
I was just a little reluctant to split those into a class hierarchy where `Credentials` and `Settings` could inherit from a say `Configuration` base class. Mostly because the multiple loading paths supported right now. Do you think we should do the classes separately?